openfeature/multi/multiprovider.go

```diff
@@ -158,39 +179,28 @@ func WithGlobalHooks(hooks ...of.Hook) Option {
 	}
 }
 
-// WithProviderHooks sets [of.Hook] instances that execute only for a specific [of.FeatureProvider]. The providerName
-// must match the unique provider name set during [multi.Provider] creation. This should only be used if you need hooks
-// that execute around a specific provider, but that provider does not currently accept a way to set hooks. This [Option]
-// can be used multiple times using unique provider names. Using a provider name that is not known will cause an error.
-func WithProviderHooks(providerName string, hooks ...of.Hook) Option {
```

Comment:
Since we're removing WithProviderHooks here, please update the doc comment of WithGlobalHooks to remove the reference to it.